### DISCUSSION TITLE
My College Does not provide Student email id , then How can apply for Github Student developer pack

### DISCUSSION THREAD

**Discussion by mystry123** [community/community#127255] [GitHub Education]
## Hi everyone,

i have all the necessary documents (e.g.  id card , fee receipts etc .) except student mail id  to proof my education institution , please provide me a way to apply for student developer pack.

**Comment by davevad93** [comment]
Hi @ritikpatel123 ,

Take a look at [this](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/) from GitHub Blog,  regarding not having an educational email it states the following:

 - `If you don’t have a school-issued email, follow the prompts to fill out additional information.You’re still eligible even if you only have a personal Gmail address, as long as you can provide alternative documentation to verify your current student status.`

**Reply by mystry123** [reply]
thanks for responding @davevad93 
the problem is here i'm not getting any kind of prompts.

<img width="1398" alt="Screenshot 2024-06-04 at 11 16 53 PM" src="https://github.com/community/community/assets/86945459/5c533834-2fca-4421-9816-e6ae8e196b8d">

**Reply by davevad93** [reply]
Hey @ritikpatel123 If no option shows up then it's probably no longer possible to get the student benefits without an educational email. In that case I think you should contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and describe your issue, maybe there's another workaround.

**Reply by Shaikh-Rayyan201** [reply]
@mystry123 hey there! did your problem got solved by raising a ticket? I'm facing the same problem.

**Reply by Abk2162** [reply]
@mystry123 even i am facing the same issue, did raising a ticket solve the problem?

**Reply by suraj-kumarprajapati** [reply]
Just put your personal mail id and college id ( id card photo) as a proof thats it

**Reply by ukcai696969** [reply]
> Just put your personal mail id and college id ( id card photo) as a proof thats it

you cant it doesn't let you continue

**Reply by suraj-kumarprajapati** [reply]
Bro recently got access

**Comment by robert-hughes7** [comment]
> ## Hi everyone,
> i have all the necessary documents (e.g. id card , fee receipts etc .) except student mail id to proof my education institution , please provide me a way to apply for student developer pack.

Did your problem get resolved?

**Comment by affan004** [comment]
i have a official uni email and id card and they still rejected me because  i happened to be on a vacation to my hometown in my country.
